example_analysis/aggregate_4/tile-based/aggregate_4_smk_test.ipynb

```diff
@@ -0,0 +1,1987 @@
+{
```

Comment:
<p>Consider keeping the names aggregate and annotate for the packages as this may be clearer.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cheeseman-lab/OpticalPooledScreens/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Yup, good point. I didn't check back about how this was used in previous examples, but we should standardize in brieflow</p>